ci/build-project.ps1

```diff
@@ -0,0 +1,63 @@
+param(
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine", ## This should ultimately resolve to "C:\b\<number>\UnrealEngine".
+    [string] $test_repo_branch,
+    [string] $test_repo_url,
+    [string] $test_repo_uproject_path,
+    [string] $test_repo_path,
+    [string] $msbuild_exe,
+    [string] $gdk_home,
+    [string] $build_platform,
+    [string] $build_state,
+    [string] $build_target
+)
+
+# Workaround for UNR-2156 and UNR-2076, where spatiald / runtime processes sometimes never close, or where runtimes are orphaned
+# Clean up any spatiald and java (i.e. runtime) processes that may not have been shut down
+Start-Process spatial "service","stop" -Wait -ErrorAction Stop -NoNewWindow
+Stop-Process -Name "java" -Force -ErrorAction SilentlyContinue
+
+# Clean up testing project (symlinks could be invalid during initial cleanup - leaving the project as a result)
```

Comment:
I'm not sure I follow, why would there be something here?

Reply:
The GDK is being symlinked into the project. To prevent a circular reference, we therefore don't want to put the project in the cloned copy of this GDK folder. As a result, the project is being put outside of the current build's directory - which is not automatically cleaned afterwards. I'm now adding cleaning to the pre-exit BK hook.